api/gen/es/rpc_connect.d.ts

```diff
@@ -239,6 +241,8 @@ export declare const GnoNativeService: {
     },
     /**
      * Make an ABCI query to the remote node.
+     * If the request path is unrecognized, return [ErrCode](#land.gno.gnomobile.v1.ErrCode).ErrUnknownRequest.
```

Comment:
Why gnomobile and not gnonative?

Reply:
Thanks for catching this. I committed a fix. (I started working on the PR a long time ago ;-)